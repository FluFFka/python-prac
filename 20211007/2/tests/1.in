486516486516486514651, -986453124561302486040685146146543
