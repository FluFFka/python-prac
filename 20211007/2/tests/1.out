986453124561789002527201632661194
